src/veh_interact.cpp

```diff
@@ -1082,7 +1082,7 @@ bool veh_interact::overview( std::function<bool(const vehicle_part &pt)> enable,
                 if( pt.ammo_current() != "null" ) {
                     auto stack = units::legacy_volume_factor / item::find_type( pt.ammo_current() )->stack_size;
                     right_print( w, y, 1, item::find_type( pt.ammo_current() )->color,
-                                 string_format( "%s  %5.1fL", item::nname( pt.ammo_current().c_str() ),
+                                 string_format( "%s  %5.1fL", item::nname( pt.ammo_current() ),
```

Comment:
I'm not sure about this one. `item::nname()` returns `std::string` which is a non-trivial type.